charts/cloudzero-agent/templates/cm.yaml

```diff
@@ -136,7 +136,7 @@ data:
           follow_redirects: true
           enable_http2: true
     remote_write:
-      - url: 'https://{{ .Values.host }}/v1/container-metrics?cluster_name={{.Values.clusterName}}&cloud_account_id={{.Values.cloudAccountId}}'
+      - url: 'https://{{ .Values.host }}/v1/container-metrics?cluster_name={{.Values.clusterName}}&cloud_account_id={{.Values.cloudAccountId | toString}}'
```

Comment:
This will coerce the value to a String, which isn't necessary since the Values Schema should prevent any non-string values from being passed in, but seems like a nice to have.